electron-app/src/services/types-common.ts

```diff
@@ -1 +1,25 @@
 export type DSRMovementType = 'withdrawal' | 'deposit';
+
+export enum Location {
+  EXTERNAL = 'external',
+  KRAKEN = 'kraken',
+  POLONIEX = 'poloniex',
+  BITTREX = 'bittrex',
+  BINANCE = 'binance',
+  BITMEX = 'bitmex',
+  COINBASE = 'coinbase',
+  TOTAL = 'total',
+  BANKS = 'banks',
+  BLOCKCHAIN = 'blockchain',
+  COINBASEPRO = 'coinbasepro',
+  GEMINI = 'gemini'
+}
+
+export interface SupportedAsset {
+  readonly active: boolean;
```

Comment:
The `active`, `ended`, `started` attributes may not exist in an asset. Is this taken into account?